unit-testing/presentation.html

```diff
@@ -1,27 +1,398 @@
+  <title>rolling-scopes-school/react-native-course - unit-testing</title>
+  <meta name="description" content="rolling-scopes-school/react-native-course - unit-testing" />
+  <meta name="apple-mobile-web-app-capable" content="yes" />
+  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="../reveal-js/dist/reset.css" />
+  <link rel="stylesheet" href="../reveal-js/dist/reveal.css" />
+  <link rel="stylesheet" href="../reveal-js/dist/theme/white.css" id="theme" />
+  <!-- Theme used for syntax highlighting of code -->
+  <link rel="stylesheet" href="../reveal-js/plugin/highlight/monokai.css" />
+  <link rel="stylesheet" href="../reveal-js/common.css" />
+  <style class="jsStyle"></style>
+</head>
+
+<body>
+  <div class="reveal">
+    <div class="slides">
+      <section class="presentation-name-slide">
+        <h4>Unit tests</h4>
+      </section>
+      <!-- 1 slide -->
+      <section>
+        <h3 class="slide-name">AGENDA</h3>
+        <ul>
+          <li class="fragment">Testing in software development</li>
+          <li class="fragment">Testing Pyramid</li>
+          <li class="fragment">Unit tests</li>
+          <li class="fragment">TDD & BDD</li>
+          <li class="fragment">Testing statistic</li>
+          <li class="fragment">Testing libraries</li>
+          <li class="fragment">Jest as a testing framework</li>
+          <li class="fragment">Husky</li>
+        </ul>
+      </section>
+      <!-- 2 slide -->
+      <section>
+        <h3 class="slide-name">Testing role in Software Development​</h3>
+        <ul>
+          <li class="fragment">Testing is about verifying that what was specified is what was delivered.</li>
+          <li class="fragment">The testing is important since it discovers defects/bugs before the delivery to the
+            client, which guarantees the quality of the software.​</li>
+        </ul>
+      </section>
+      <!-- 3 slide -->
+      <section>
+        <h3 class="slide-name">Major benefits of software testing​</h3>
+        <ul>
+          <li class="fragment">Launch a flawless software application</li>
+          <li class="fragment">Prevent targeted malware attacks</li>
+          <li class="fragment">Improves the user experience</li>
+          <li class="fragment">Curtail maintain costs</li>
+          <li class="fragment">Improve time-to-market</li>
+          <li class="fragment">Enhance quality of apllication</li>
+          <li class="fragment">Boost competitiveness</li>
+        </ul>
+      </section>
+      <!-- 4 slide -->
+      <section>
+        <h3 class="slide-name">The scope of Testing​</h3>
+        <ul>
+          <li class="fragment"> The test scope defines what areas, features, and functionalities of an application
+            should be tested</li>
+        </ul>
+      </section>
+      <!-- 5 slide -->
+      <section>
```

Comment:
class="picture-slide" and remove image height attribute

Reply:
done1